packages/react-native-worklets/android/CMakeLists.txt

```
@@ -105,7 +105,11 @@ else()
 endif()
 
 if(${JS_RUNTIME} STREQUAL "hermes")
-  target_link_libraries(worklets hermes-engine::libhermes)
+  if(ReactAndroid_VERSION_MINOR GREATER_EQUAL 82)
+    target_link_libraries(worklets hermes-engine::libhermesvm)
```

```suggestion
    target_link_libraries(worklets hermes-engine::hermesvm)
```

I think the target name is just `hermesvm`

---

Thanks! Fixed in 97a685ee5cdcb6665b36fec1a03e89363e7c9c8f.